ml-agents/mlagents/trainers/tests/test_bcmodule.py

```diff
@@ -31,7 +31,7 @@ def ppo_dummy_config():
         use_recurrent: false
         memory_size: 8
         pretraining:
-          demo_path: ./demos/ExpertPyramid.demo
```

Comment:
I think we can change this to anything as the demo_path is simply overridden in the test